packages/bun-types/bun-test.d.ts

```diff
@@ -1,82 +1,433 @@
+     * Asserts that a value is "truthy".
+     *
+     * To assert that a value equals `true`, use `toBe(true)` instead.
+     *
+     * @link https://developer.mozilla.org/en-US/docs/Glossary/Truthy
+     * @example
+     * expect(true).toBeTruthy();
+     * expect(1).toBeTruthy();
+     * expect({}).toBeTruthy();
+     */
     toBeTruthy(): void;
+    /**
+     * Asserts that a value is "falsy".
+     *
+     * To assert that a value equals `false`, use `toBe(false)` instead.
+     *
+     * @link https://developer.mozilla.org/en-US/docs/Glossary/Falsy
+     * @example
+     * expect(true).toBeTruthy();
+     * expect(1).toBeTruthy();
+     * expect({}).toBeTruthy();
+     */
     toBeFalsy(): void;
+    /**
+     * Asserts that a value is defined. (e.g. is not `undefined`)
+     *
+     * @example
+     * expect(true).toBeDefined();
+     * expect(undefined).toBeDefined(); // fail
+     */
     toBeDefined(): void;
+    /**
+     * Asserts that a value is `undefined`.
+     *
+     * @example
+     * expect(undefined).toBeUndefined();
+     * expect(null).toBeUndefined(); // fail
+     */
     toBeUndefined(): void;
-    toBeNaN(): void;
+    /**
+     * Asserts that a value is `null`.
+     *
+     * @example
+     * expect(null).toBeNull();
+     * expect(undefined).toBeNull(); // fail
+     */
     toBeNull(): void;
-    toBeGreaterThan(value: number | bigint): void;
-    toBeGreaterThanOrEqual(value: number | bigint): void;
-    toBeLessThan(value: number | bigint): void;
-    toBeLessThanOrEqual(value: number | bigint): void;
-    toThrow(error?: string | Error | ErrorConstructor | RegExp): void;
+    /**
+     * Asserts that a value can be coerced to `NaN`.
+     *
+     * Same as using `Number.isNaN()`.
+     *
+     * @example
+     * expect(NaN).toBeNaN();
+     * expect(Infinity).toBeNaN();
+     * expect("notanumber").toBeNaN();
+     */
+    toBeNaN(): void;
+    /**
+     * Asserts that a value is a `number` and is greater than the expected value.
+     *
+     * @example
+     * expect(1).toBeGreaterThan(0);
+     * expect(3.14).toBeGreaterThan(3);
+     * expect(9).toBeGreaterThan(9); // fail
+     *
+     * @param expected the expected number
+     */
+    toBeGreaterThan(expected: number | bigint): void;
+    /**
+     * Asserts that a value is a `number` and is greater than or equal to the expected value.
+     *
+     * @example
+     * expect(1).toBeGreaterThanOrEqual(0);
+     * expect(3.14).toBeGreaterThanOrEqual(3);
+     * expect(9).toBeGreaterThanOrEqual(9);
+     *
+     * @param expected the expected number
+     */
+    toBeGreaterThanOrEqual(expected: number | bigint): void;
+    /**
+     * Asserts that a value is a `number` and is less than the expected value.
+     *
+     * @example
+     * expect(-1).toBeLessThan(0);
+     * expect(3).toBeLessThan(3.14);
+     * expect(9).toBeLessThan(9); // fail
+     *
+     * @param expected the expected number
+     */
+    toBeLessThan(expected: number | bigint): void;
```

Comment:
can we check that bigint works here

i don't remember if it does

Reply:
Checked with bigint, it works.